app/controllers/api/available_slots_controller.rb

```diff
@@ -3,14 +3,15 @@ class AvailableSlotsController < ApiController
     def index
       prison = Prison.find(params.fetch(:prison_id))
 
-      slot_availability = ApiSlotAvailability.new(
-        prison: prison,
-        use_nomis_slots: false
-      )
-      slot_availability.restrict_by_prisoner(
-        prisoner_number: params.fetch(:prisoner_number),
-        prisoner_dob: Date.parse(params.fetch(:prisoner_dob))
-      )
+      slot_availability = ApiSlotAvailability.new(prison: prison,
+                                                  use_nomis_slots: false)
+
+      Timebox.new(TIMEBOX_LIMIT).run do
```

Comment:
If you look at the block being passed in, it calls `restrict_by_prisoner`, a command style method (rather than a query) - no return result, it just populates `slots` in the `slot_availability` instance. If this times out we don't need to return anything - `slots` just doesn't get populated and retains the vanilla info that it acquired during `initialize`.